DEPENDENCIES.md

```diff
@@ -61,28 +61,53 @@ Bitcoin needs to be running with the following options enables:
 - `txindex` to be able to look for non-wallet transactions
 - `server` to run rpc commands
 
-Here's an example of a `bitcoin.conf` you can use:
+Here's an example of a `bitcoin.conf` you can use for mainnet:
 
 ```
-# [debug]
-regtest=1
-daemon=1
-debug=1
-logips=1
-
 # [rpc]
 server=1
 rpcuser=user
 rpcpassword=passwd
+rpcservertimeout=600
+
+# [zmq]
+zmqpubhashblock=tcp://127.0.0.1:28332
+zmqpubrawblock=tcp://127.0.0.1:28332
+zmqpubhashtx=tcp://127.0.0.1:28333
+zmqpubrawtx=tcp://127.0.0.1:28333
 
 # [blockchain]
 txindex=1
 
-# [ZMQ]
+# [others]
+daemon=1
+debug=1
+maxtxfee=1
+```
+
+And here's an example for regtest:
```

Comment:
Remove the regtest example. It's mainly the same just adding regtest=1. The mainnet one should be enough

Reply:
Done in ada9645.